### DISCUSSION TITLE
New Tubefeeder Icon

### DISCUSSION THREAD

**Discussion by daudix** [Tubefeeder/Pipeline#65] [Q&A] [answered]
Here is some variants:

# 1.

![Tubefeeder](https://user-images.githubusercontent.com/77155297/174449731-9b7086e5-c7bd-4a6e-8a80-3649b56acf22.png)

# 2. TV

![Tubefeeder TV](https://user-images.githubusercontent.com/77155297/174449749-f56e8971-32e1-4be9-9976-c077063644b2.png)

# 3. Neo

![Tubefeeder Neo](https://user-images.githubusercontent.com/77155297/174449747-462004ad-e5d6-4ade-83ff-b76eabbacf3e.png)

# 4. TV Neo

![Tubefeeder TV Neo](https://user-images.githubusercontent.com/77155297/174449750-44b24d19-8320-4d45-855a-d2411a511624.png)

# 5. Alt

![Tubefeeder Alt](https://user-images.githubusercontent.com/77155297/174449733-d9f41972-3f62-491d-9a09-f1600314fbbf.png)

# 6. Alt TV

![Tubefeeder Alt TV](https://user-images.githubusercontent.com/77155297/174449737-7cf0b247-20b2-4402-ac4e-78dfd61ca3dd.png)

# 7. Alt Neo

![Tubefeeder Alt Neo](https://user-images.githubusercontent.com/77155297/174449735-388c7d90-5388-43f1-9799-aeea72f39f25.png)

# 8. Alt TV Neo

![Tubefeeder Alt  TV Neo](https://user-images.githubusercontent.com/77155297/174449738-8454e9c1-4dca-45b2-9547-982827324558.png)

# 9. Gradient

![Tubefeeder Gradient](https://user-images.githubusercontent.com/77155297/174449741-469045e2-19c2-4b6f-bd99-e523e6ffec28.png)

# 10. Gradient TV

![Tubefeeder Gradient TV](https://user-images.githubusercontent.com/77155297/174449744-47d98e4f-a75d-4cb3-911a-0bd0ff3391a1.png)

# 11. Gradient Neo

![Tubefeeder Gradient Neo](https://user-images.githubusercontent.com/77155297/174449743-27b11045-8bb5-48d8-90cb-c383a21ecf69.png)

# 12. Gradient TV Neo

![Tubefeeder Gradient TV Neo](https://user-images.githubusercontent.com/77155297/174449746-da1ea2c3-b9a5-4510-9add-25dc1e0a8ac1.png)

# 13. Classic

![Tubefeeder Classic](https://user-images.githubusercontent.com/77155297/174449739-db5b9f8a-22f9-4733-9f06-31bc87221785.png)

# 14. Classic Neo

![Tubefeeder Classic Neo](https://user-images.githubusercontent.com/77155297/174449740-39967ec8-5804-42b1-998b-e6d123f3d466.png)

**Comment by daudix** [comment]
@Schmiddiii Take a Look ;)

**Comment by Schmiddiii** [comment]
I will discuss this a little bit in the [Matrix Group](https://matrix.to/#/%23tubefeeder:matrix.org?via=matrix.org). This will maybe take a while, but thanks for all these nice icons.

**Reply by daudix** [reply]
Thank you, sorry for a lot of variants 😅

**Reply by daudix** [reply]
Added a "Neo" version, classic + new

**Comment by Schmiddiii** [comment, marked as answer]
The matrix channel decided on the new "Classic" icon. It will be used in a (probably soon) 1.8.0-update of Tubefeeder.
